microsetta_public_api/api/microsetta_public_api.yml

```diff
@@ -619,6 +619,555 @@ paths:
         '404':
           $ref: '#/components/responses/404NotFound'
 
+  '/{dataset}/plotting/diversity/alpha/{alpha_metric}/percentiles-plot':
+    parameters:
+      - $ref: '#/components/parameters/namedDataset'
+      - $ref: '#/components/parameters/alphaMetric'
+      - $ref: '#/components/parameters/percentiles'
+      - $ref: '#/components/parameters/sampleIdQueryOpt'
+    get:
+      operationId: microsetta_public_api.api.plotting.plot_alpha_filtered_alt
+      tags:
+        - Plotting
+        - Alpha Diversity
+      summary: Get a Vega schema for alpha diversity CDF matching criteria
+      description: Get a Vega schema for alpha diversity cdf matching criteria
+      parameters:
+        - in: query
+          name: age_cat
+          schema:
+            type: string
+            example: '30s'
+        - in: query
+          name: bmi_cat
+          schema:
+            type: string
+            example: 'Normal'
+      responses:
+        '200':
+          description: Successfully returned Vega JSON
+          content:
+            application/json:
+              schema:
+                type: object
+                additionalProperties: true
+        '404':
+          $ref: '#/components/responses/404NotFound'
+        '422':
+          $ref: '#/components/responses/422UnprocessableEntity'
+    post:
+      operationId: microsetta_public_api.api.plotting.plot_alpha_filtered_json_query_alt
+      tags:
+        - Plotting
+        - Alpha Diversity
+      summary: Get a Vega schema for alpha diversity cdf matching criteria
+      description: Get a Vega schema for alpha diversity cdf matching criteria
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/metadataQuery'
+      responses:
+        '200':
```

Comment:
It looks like this response structure is replicated, should it be a `$ref`?

Reply:
Eh yeah you're right. There's probably about two dozen other repeated segments throughout here too. I might just be happy to kick the can down the road to a maintenance PR.

Reply:
Fixed in https://github.com/biocore/microsetta-public-api/pull/58/commits/b6060795c459dd3d2ec76223c64a369deb2afce0

Reply:
Okay cool, thanks!